tests/components/main.test.tsx

```diff
@@ -15,21 +15,36 @@ describe('Main component shows the page', () => {
     }
     ;(useRouter as jest.Mock).mockReturnValue(mockRouter)
 
-    render(<Main />)
+    render(<Main loading={false} />)
     expect(screen.queryByText('UI Online')).toBeInTheDocument()
     expect(screen.queryByText('Benefit Year')).toBeInTheDocument()
     expect(screen.queryByText('Claim Status')).toBeInTheDocument()
   })
 })
 
+describe('Main component shows loading', () => {
+  it('has our title, details, and body with shimmer', () => {
+    const mockRouter = {
+      locale: 'en',
+    }
+    ;(useRouter as jest.Mock).mockReturnValue(mockRouter)
```

Comment:
Do you know why these lines begin with a semicolon? I see the pattern exists below too.

Reply:
I do not remember! I'm duping the pattern above and I don't recall why it looks like this

Reply:
taking away the semicolon gives you the error:

```
[tsserver 2349] [E] This expression is not callable.                                                                                   
   Type '{ locale: string; }' has no call signatures.
```

which makes me suspect this is Jest Mocks specific syntax magic